test-cases/performance/perf-regression-latency-250gb-with-nemesis.yaml

```diff
@@ -16,6 +16,7 @@ n_monitor_nodes: 1
 instance_type_loader: 'c5.2xlarge'
 instance_type_monitor: 't3.large'
 instance_type_db: 'i3.2xlarge'
+max_events_severities: 'DatabaseLogEvent.REACTOR_STALLED=STARRED'
```

Comment:
it will limit the severity of `DatabaseLogEvent.REACTOR_STALLED` to be maximum `STARRED`

Reply:
this mean it would fail the test ?

Reply:
why this is needed ?

Reply:
this is needed, to during a specific test, be able to mark specific events (in this case `DatabaseLogEvent.REACTOR_STALLED`) to have a maximum severity of `STARRED`.
It doesn't affect any other event, nor any other test than the one using this parameter, nor it will fail the current test when this event occurs (we had in one of the latest jobs 4500 `DatabaseLogEvent.REACTOR_STALLED`